# data
SingleJointPosition base
SingleJointPosition shoulder
SingleJointPosition elbow
SingleJointPosition wrist
GripperPosition gripper
SingleJointPosition wristRotate